mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/logging/impl/Log.java

```diff
@@ -134,4 +136,12 @@
 	@Message(id = 1237, value = "Invalid GeoJSON type %1$s. Expecting %2$s.")
 	HibernateException invalidGeoJsonType(String actualType, String expectedType);
 
+	@Message(id = 1238, value = "Unable to execute command \"%s\". Error message : %s. Code name: %s")
+	HibernateException unableToExecuteCommand(String command, String errorMessage,String codeMessage, @Cause Exception e);
+
+	@Message(id = 1239, value = "Dialect %s does not support named parameters when calling stored procedures")
+	HibernateException dialectDoesNotSupportNamedParametersForStoredProcedures( @FormatWith(ClassObjectFormatter.class) Class<?> dialectClass );
+
+	@Message(id = 1240, value = "Procedures returning muliple documents are not supported. Procedure '%1$s' returned %2$d results")
+	HibernateException multipleDocumentReturnedByStoredProcedure(String storedProcedureName, int size);
```

Comment:
We usually have a . at the end of the log messages (this remark is for all the messages above).

+ s/muliple/multiple/

Reply:
@DavideD the s/muliple/multiple/ change was not done.